src/client/components/pages/parts/delete-collection-modal.js

```diff
@@ -0,0 +1,117 @@
+import * as bootstrap from 'react-bootstrap';
+import {FontAwesomeIcon} from '@fortawesome/react-fontawesome';
+import PropTypes from 'prop-types';
+import React from 'react';
+import request from 'superagent';
+
+
+const {Alert, Button, Modal} = bootstrap;
+
+class DeleteCollectionModal extends React.Component {
+	constructor(props) {
+		super(props);
+		this.state = {
+			error: null,
+			show: this.props.show
+		};
+
+		this.handleClose = this.handleClose.bind(this);
+		this.handleDelete = this.handleDelete.bind(this);
+	}
+
+	componentDidUpdate(prevProps) {
+		if (prevProps.show !== this.props.show) {
+			// eslint-disable-next-line react/no-did-update-set-state
+			this.setState({show: this.props.show});
+		}
+	}
+
+	handleClose() {
+		this.setState({show: false}, this.props.closeDeleteModalCallback);
+	}
+
+	handleDelete() {
+		request.post(this.deleteUrl)
+			.send()
+			.then((res) => {
+				window.location.href = '/';
+			}, (error) => {
+				this.setState({
+					error: 'Internal Error'
+				});
+			});
+	}
+
+	render() {
+		const {collection} = this.props;
+		this.collectionUrl = `/collection/${collection.id}`;
+		this.deleteUrl = `${this.collectionUrl}/delete/handler`;
+
+		let errorComponent = null;
+		if (this.state.error) {
+			errorComponent =
+				<Alert bsStyle="danger">{this.state.error}</Alert>;
+		}
+
+		return (
+			<Modal
+				show={this.state.show}
+				onHide={this.handleClose}
+			>
+				<Modal.Header closeButton>
+					<Modal.Title>Confirm Deletion</Modal.Title>
+				</Modal.Header>
+				<Modal.Body>
+					<Alert bsStyle="warning">
+						<h4>
+							<FontAwesomeIcon icon="exclamation-triangle"/>&nbsp;
+							You’re about to delete the Collection - {collection.name}.
+						</h4>
+					</Alert>
+					<p>
+						Make sure you actually want to delete this Collection <br/>
+						There is no way to undo this.
+					</p>
+					{errorComponent}
+				</Modal.Body>
+				<Modal.Footer>
+					<Button onClick={this.handleClose}>
```

Comment:
```suggestion
					<Button bsStyle="info" onClick={this.handleClose}>
```

I would also suggest removing the `times-circle` icon entirely, just so that it stays absolutely clear which button actually deletes the collection.  What do you think?

<img width="614" alt="Capture d’écran 2020-06-25 à 12 15 00" src="https://user-images.githubusercontent.com/6179856/85701329-901bf380-b6dd-11ea-8f8c-a3b8369b8171.png">